reactor-core/src/main/java/reactor/core/publisher/Traces.java

```diff
@@ -80,7 +80,7 @@
 						                                                      .newInstance();
 						return Stream.of(function);
 					}
-					catch (Throwable e) {
+					catch (ReflectiveOperationException | LinkageError e) {
```

Comment:
a brief analysis of constructors shows that other types of exceptions are unlikely, but I'd prefer to add the catch-all clause back at the end still (in case said constructors "evolve")

Reply:
also can you put up a comment that this catch clause is there to help with static analysis tools that warn about potential `LinkageError` and such?

Reply:
Sure 👍